src/web_app/js/peerconnectionclient.js

```diff
@@ -29,6 +29,22 @@ var PeerConnectionClient = function(params, startTime) {
     '  constraints: \'' + JSON.stringify(params.peerConnectionConstraints) +
     '\'.');
 
+  if (typeof RTCPeerConnection.generateCertificate === 'function') {
+    trace('Trying to generate an ECDSA certificate.');
+    RTCPeerConnection.generateCertificate({name: 'ECDSA', namedCurve: 'P-256'})
+    .then(function(cert) {
+      if (cert.expires) {
+        trace('Certificate generated successfully.');
+        params.peerConnectionConfig.certificates = [cert];
+      } else {
```

Comment:
In M49 we had not implemented the expires attribute, now we have. There will always be an expires attribute; you can remove the if-else here.